WORKSPACE

```diff
@@ -203,8 +203,7 @@ KOTLINC_RELEASE = {
 kt_download_local_dev_dependencies()
 
 kotlin_repositories(compiler_release = KOTLINC_RELEASE)
-
-register_toolchains("//third_party/java/arcs/build_defs/internal:kotlin_toolchain")
```

Comment:
I am not familiar with how these directories work, but do you want to remove that one if it is now unused? or why didn't you change the version number there?

Reply:
The definition of a toolchain has to occur in a BUILD file, it cannot occur in WORKSPACE. 

The use of a toolchain  (register_toolchains) has to happen in the WORKSPACE file.  This is pretty much what the official README recommends to do.  I dunno what happens if you have two register_toolchains() of kotlin toolchains, it might be ambiguous, or the second one might win?

It's more certain to do it this way and follow the official docs.

Reply:
Right. But is "third_party/java/arcs/build_defs/internal:kotlin_toolchain" now unused? if so, can we remove it from that file (arcs/third_party/java/arcs/build_defs/internal/BUILD)?
If it is still used somewhere else (I don't think so), should we update that one as well?

Reply:
oh, good catch, I'm blind, I was reading that as a reference to the internal:BUILD that ships with the rules_kotlin lib, I didn't notice we already had a custom toolchain at that location. I can just update that one.  let me revert and push